### ISSUE TITLE
Stepping down / abandoning Mailtrain

### ISSUE THREAD

**Issue by andris9** [Mailtrain-org/mailtrain#372]
@gsiou  @HLFH @witzig @gkrol-zermelo @brentdur @MelwinKfr @programster @mystvearn147 @wotomg @theyough @killua-eu @bures

Hi, I marked here all users who have commited to Mailtrain in the last 6 months or so.

I guess that anyone who has kept an eye on the Mailtrain project has noticed the piling issues and zero commits. This is mostly because I have spent all my time on other projects (mostly on [WildDuck](https://github.com/nodemailer/wildduck)) and do not see myself into commiting to Mailtrain again.

So, in short, i'm abandoning the project. If anyone is interested in taking over the ownership then let me know in the comments of this issue. I can give admin rights to the Mailtrain-org Github organization and also transfer the mailtrain.org domain name to the new owner. If there's no interest then I'll add a notice to the README and mailtrain.org that the project is abandoned and first to claim would get the ownership.

Thanks,
Andris

**Comment by gkrol-zermelo** [comment]
Hi! We're using Mailtrain for sending our customers update notices and other news, and for now it does exactly what we want to. We could use some minor usability improvements, but for now its "good enough". We really like the fact that it's self-hosted. That makes it a lot easier to comply with (EU) data protection laws.

I suggest we run Mailtrain in "please send pull requests" mode. I'd be available to review pull requests, but would not be interested in the mailtrain.org domain name. I suppose we could just have it redirect to github or have it expire.

Thanks for your work, and good luck with your other projects!

**Comment by programster** [comment]
Really sorry to hear that @andris9 but can completely understand. Maintaining a popular project on Github is a lot of work! It is a great piece of software, and it could probably run for years to come with just "maintenance" updates rather then new features etc. I'm really glad it exists as self-hosting solves a lot of potential issues like @gkrol-zermelo mentioned.

**Comment by bures** [comment]
Sorry to hear, but I completely understand. I would be actually very happy if we could continue with Mailtrain. We are gradually introducing it to deliver emails within our international NGO. 

As for the future enhancements to Mailtrain, I'm also working on a new version of Mailtrain that introduces React-based UI, granular permissions, user roles and some more enhancements. Furthermore on the server it introduces DB access using Knex and rewrite of most of callback-based code to async/await, which simplifies to code at some places quite significantly. The work in progress is here https://github.com/bures/mailtrain/tree/access.

Unfortunately, I'm quite busy at work, so the progress is very slow. However, I hope to be able to make a release candidate in summer.

That said, I would be happy to remain part of the core contributors as our stakes in Mailtrain are quite high.

@andris9, thank you very much for creating this awesome project.

**Comment by mailanetworks** [comment]
Hi, this is really sad news! We also use it for internal communications and it works pretty well. I hope the project finds a new owner, and @andris9 thank you for the project and good luck!

**Comment by mailfromabox** [comment]
Should make a hosted paid version on the domain... could fund the development.

**Comment by bures** [comment]
My intentions are to contribute under open-source license because I'm doing the extensions for our NGO on voluntary basis anyway. However, I would not oppose is anyone wants to invest more time maintenance and pays for it by hosting a paid version as a service. Anyway, once (if) I complete our extensions regarding the granular permissions and hierarchical namespaces, it should definitely help creating a multi-tenant hosted solution.

**Comment by killua-eu** [comment]
I'm using maitrain also for our NGO (different one then @bures). I'm too unfamiliar with js to be of any real help with ongoing development of mailtrain, but I maintain several Ansible roles, that I use to deploy mailtrain, backups and stuff. If there's a dev among us who'd like to pick up mailtrain and would like to use a hosted service to support the development, I can definitely volunteer to set up the infrastructure - just let me know so that we can discuss implementation details.

**Comment by andris9** [comment]
Thanks for the responses.

* I sent out invitations to @gkrol-zermelo and @bures to get access to the Mailtrain Github organization. You can then decide yourself what you do with that access, do you just accept PR's or give someone else access or do nothing etc.
* I redirected mailtrain.org domain name to the Github project page and also removed MX records
* Disabled auto-renewal for mailtrain.org domain name, it should expire by April

So I guess for me that is all for now regarding Mailtrain.

**Comment by andris9** [comment]
Oh, just realized I sent the invitations for this particular repo and not for the origanization. Resending...

**Comment by bures** [comment]
@andris9 thanks a lot once again for everything!

I'll be moving this discussion to another thread on Mailtrain future (#373). Please follow up there.

**Comment by brentdur** [comment]
A bit late to the thread here, but just wanted to say thanks @andris9 for creating this project. It's been invaluable to a number of non-profits I work with in managing their mailing lists. Best of luck in future endeavors
